package.json

```diff
@@ -40,6 +40,7 @@
   },
   "dependencies": {
     "bootstrap": "4.1.x",
-    "jquery": "3.3.x"
+    "jquery": "3.3.x",
+    "popper.js": "1.14.x"
```

Comment:
Webpack.config.js - please remove:

```js
{
        from: path.resolve(__dirname, 'src/files'),
        to: path.resolve(__dirname, 'dist/files')
      }
```

it causes that problem: `ERROR in [copy-webpack-plugin] unable to locate '/Users/michal/dev/www/codeloop-website/src/files' at '/Users/michal/dev/www/codeloop-website/src/files'
C`

- [ ] src/files